src/microbe_stage/MicrobeAI.cs

```diff
@@ -73,6 +73,9 @@ public class MicrobeAI
     [JsonIgnore]
     private MicrobeSignalCommand receivedCommand = MicrobeSignalCommand.None;
 
+    [JsonIgnore]
+    private bool hasBeenNearPlayer;
```

Comment:
Shouldn't this be saved? If an old save doesn't have it, this will be false. But for newer saves this seems like a good thing to save.